src/lighteval/tasks/default_tasks.py

```diff
@@ -6619,21 +6621,27 @@
     trust_dataset=True,
     version=0,
 )
-coqa_lighteval = LightevalTaskConfig(
-    name="coqa",
-    suite=["lighteval"],
-    prompt_function=prompt.coqa,
-    hf_repo="coqa",
+coqa_first_question = LightevalTaskConfig(
```

Comment:
For all evals in this file, either don't remove the original version (so we keep lighteval + yours) or keep the same name and change the version to 1.

Reply:
The original version didn't make any sense; each consecutive question can depend on each other. So I think this should be really the "first"version. Same for drop, it was outright broken iirc

Reply:
This is a matter of reproducibility/continuity in the eval suite: if people search for these tasks in the future they will have completely disappeared - if you just say your version is v1, it makes it easy for people to update and see what to replace things by

Reply:
Fixed it. Can I merge @clefourrier

Reply:
I must be blind but I still see the name as `coqa_first_question` not the original `coqa_lighteval`

Reply:
maybe I'm not looking at the correct commit though, so if this is fixed feel free to merge

Reply:
That's just variable name....